Sources/App/Migrations/039/UpdateSearch4.swift

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020-2021 Dave Verwer, Sven A. Schmidt, and other contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Fluent
+import SQLKit
+
+
+struct UpdateSearch4: Migration {
+    let dropSQL: SQLQueryString = "DROP MATERIALIZED VIEW search"
+    
+    func prepare(on database: Database) -> EventLoopFuture<Void> {
+        guard let db = database as? SQLDatabase else {
+            fatalError("Database must be an SQLDatabase ('as? SQLDatabase' must succeed)")
+        }
+        
+        return db.raw(dropSQL).run()
+            .flatMap { db.raw("""
+            CREATE MATERIALIZED VIEW search AS
+            SELECT
+              p.id AS package_id,
+              p.score,
+              r.keywords,
+              r.last_commit_date,
+              r.license,
+              r.name AS repo_name,
+              r.owner AS repo_owner,
+              r.stars,
+              r.last_activity_at,
+              r.summary,
+              v.package_name,
+              v.resolved_dependencies
```

Comment:
I think we need to have a think about what the plans are for searching through a packages dependencies. Or rather: given that we're only looking at the count at this time, I think we want to store a dependency count only at this time instead of copying the whole column, which can be quite big.

Reply:
On second thought, I think we should hold off on this filter until we've got some usage info regarding the filters in general. The others have all been "client side" filter logic but before we start making further schema changes we should evaluate where we are.

Would you mind parking it in the meantime, James?